### DISCUSSION TITLE
allure-results does not exist

### DISCUSSION THREAD

**Discussion by piyush466** [allure-framework/allure2#2472] [Questions & Support]
'allure' is not recognized as an internal or external command,
operable program or batch file.
Build step 'Execute Windows batch command' marked build as failure
[Accounting] $ C:\Users\ASUS\Downloads\allure-2.27.0\allure-2.27.0\bin\allure.bat generate -c -o C:\ProgramData\Jenkins\.jenkins\workspace\Accounting\allure-report
allure-results does not exist
Report successfully generated to C:\ProgramData\Jenkins\.jenkins\workspace\Accounting\allure-report
Allure report was successfully generated.
Creating artifact for the build.
Artifact was added to the build.
Finished: FAILURE

**Comment by baev** [comment]
The issue is that you need to specify the path to `allure-results`. See [docs](https://allurereport.org/docs/integrations-jenkins/) for the details.
